### PR TITLE
Enable ipytest test_slice

### PR DESCRIPTION
**Pull request by jameslan**

### CODE REVIEW

File: Tests/test_slice.py [hunk 1]
```diff
@@ -2830,40 +2839,40 @@ def __delitem__(self, index):
...
+        del foo()[2<<64:]
+        self.assertEqual(setVal, slice(36893488147419103232, None))
+        del foo()[:2]
+        self.assertEqual(setVal, slice(2))
+        del foo()[:2<<64]
+        self.assertEqual(setVal, slice(36893488147419103232))
+        del foo()[2:3]
+        self.assertEqual(setVal, slice(2, 3))
+        del foo()[2<<64:3<<64]
+        self.assertEqual(setVal, slice(36893488147419103232, 55340232221128654848))
```
Comment (slozier) [inline Tests/test_slice.py:2857]: long versions:
```Python
        del foo()[long(2):]
        self.assertEqual(setVal, slice(2, None))
        del foo()[:long(2)]
        self.assertEqual(setVal, slice(2))
        del foo()[long(2):long(3)]
        self.assertEqual(setVal, slice(2, 3))
```

File: Tests/test_slice.py [hunk 2]
```diff
@@ -2663,46 +2672,46 @@ def __delslice__(self, i, j, value):
...
     def test_getslice_missing_values(self):
         # missing values are different from passing None explicitly
         class myint(int): pass
         class mylong(long): pass
-        
+
         class foo:
-            def __getslice__(self, i, j):
+            def __getslice__(self, i, j):  # shouldn't be called
                 return (i, j)
```
Comment (slozier) [inline Tests/test_slice.py:2688]: If it shouldn't be called, maybe raise an exception? I'm also fine with just removing `__getslice__`.

Comment (jameslan) [reply Tests/test_slice.py]: Removed `__getslice__`

File: Src/IronPythonTest/Cases/IronPythonCasesManifest.ini [hunk 3]
```diff
@@ -12,9 +12,6 @@ Reason=Blocked by https://github.com/IronLanguages/ironpython3/issues/541
 RunCondition=NOT $(IS_POSIX)
 Reason=Blocked by https://github.com/IronLanguages/ironpython3/issues/541
 
-[IronPython.test_class]
```
[the remark this answers is not in the document]
Comment (jameslan) [reply Src/IronPythonTest/Cases/IronPythonCasesManifest.ini]: Oops... Fixed.

File: Tests/test_slice.py [hunk 4]
```diff
@@ -2663,46 +2672,46 @@ def __delslice__(self, i, j, value):
...
-        validate_slice_result(foo()[:True], (0, 1))
-        validate_slice_result(foo()[False:True], (0, 1))
-        
+        validate_slice_result(foo()[:], slice(None))
+        validate_slice_result(foo()[2:], slice(2, None))
+        validate_slice_result(foo()[2<<64:], slice(36893488147419103232, None))
+        validate_slice_result(foo()[:2], slice(2))
+        validate_slice_result(foo()[:2<<64], slice(36893488147419103232))
+        validate_slice_result(foo()[2:3], slice(2, 3))
+        validate_slice_result(foo()[2<<64:3<<64], slice(36893488147419103232, 55340232221128654848))
```
[the remark this answers is not in the document]
Comment (jameslan) [reply Tests/test_slice.py:2702]: Updated

File: Tests/test_slice.py [hunk 5]
```diff
@@ -2742,41 +2751,41 @@ def __setitem__(self, index, value):
...
+        foo()[2<<64:] = 123
+        self.assertEqual(setVal, (slice(36893488147419103232, None), 123))
+        foo()[:2] = 123
+        self.assertEqual(setVal, (slice(2), 123))
+        foo()[:2<<64] = 123
+        self.assertEqual(setVal, (slice(36893488147419103232), 123))
+        foo()[2:3] = 123
+        self.assertEqual(setVal, (slice(2, 3), 123))
+        foo()[2<<64:3<<64] = 123
+        self.assertEqual(setVal, (slice(36893488147419103232, 55340232221128654848), 123))
```
[the remark this answers is not in the document]
Comment (jameslan) [reply Tests/test_slice.py:2767]: Updated.

File: Tests/test_slice.py (hunk 1 above)
Comment (jameslan) [reply Tests/test_slice.py:2855]: Updated

**Pull request merged**